client/packages/requisitions/src/RnRForms/DetailView/SidePanel.tsx

```diff
@@ -53,6 +63,31 @@ export const SidePanel = ({ data }: { data: RnRFormFragment }) => {
             <PanelLabel>{t('label.created')}</PanelLabel>
             <PanelField>{localisedDate(data.createdDatetime)}</PanelField>
           </PanelRow>
+          <PanelRow>
+            <PanelLabel flex={0.4}>{t('heading.reference')}</PanelLabel>
```

Comment:
```suggestion
            <PanelLabel flex={0.5}>{t('heading.reference')}</PanelLabel>
```

Not sure why you were using 0.4? But anyways, it makes the 2 labels unalign when you change screen size 👀

Reply:
lol no idea, probably typo somehow 😁